integration/projects_resources_get_test.go

```diff
@@ -246,7 +246,8 @@ ID                                      IP                 Name             Stat
       ],
       "features": [ "install_agent" ],
       "available": true
-    },
+	},
+	"vpc_uuid": "00000000-0000-4000-8000-000000000000",
```

Comment:
Same here, this is indented slightly more than the other lines.

Reply:
Yea, that's actually part of a multiline string.  Looks like I had indentation set to use tabs.